tracim/tracim/lib/notifications.py

```diff
@@ -210,6 +211,24 @@ def _get_sender(self, user: User=None) -> str:
             email_address = email_address
         )
 
+    @staticmethod
+    def _log_notification(
+            recipient: str,
+            subject: str
+    ) -> None:
```

Comment:
According to our live-talk, add a parameter called `action` which would be "CREATED" or "SENT".

Then, this parameter will be set to "CREATED" in the log where you already log data, and add a call to `_log_notification` with action "SENT" in `async_email_sender.send_mail` function.

Let's talk if its not clear for you.